src/current/v25.2/create-policy.md

```diff
@@ -53,20 +53,25 @@ The `USING` and `WITH CHECK` expressions can reference table columns and use ses
-| `UPDATE`                            | ✓                                                   | —                                                | ✓                                                 | ✓                                                     | —                                             |
-| `DELETE`                            | ✓                                                   | —                                                | —                                                 | —                                                     | ✓                                             |
-| `INSERT ... ON CONFLICT DO UPDATE`  | ✓                                                   | —                                                | ✓                                                 | ✓                                                     | —                                             |
-| `UPSERT`                            | ✓                                                   | —                                                | ✓                                                 | ✓                                                     | —                                             |
+| Command / clause pattern             | `SELECT` policy - `USING` (row that already exists) | `SELECT` policy - `USING` (row being added) | `INSERT` policy - `WITH CHECK` (row being added) | `UPDATE` policy - `USING` (row before the change) | `UPDATE` policy - `WITH CHECK` (row after the change) | `DELETE` policy - `USING` (row to be removed) |
+|--------------------------------------|-----------------------------------------------------|---------------------------------------------|--------------------------------------------------|---------------------------------------------------|-------------------------------------------------------|-----------------------------------------------|
+| `SELECT`                             | ✓                                                   | —                                           | —                                                | —                                                 | —                                                     | —                                             |
+| `SELECT ... FOR UPDATE / FOR SHARE`  | ✓                                                   | —                                           | —                                                | ✓                                                 | —                                                     | —                                             |
+| `INSERT`                             | —                                                   | ✓                                           | ✓                                                | —                                                 | —                                                     | —                                             |
```

Comment:
I just noticed that we don't apply SELECT policies on this INSERT (3rd column). It can only be applied in the INSERT ... RETURNING case, and only if the column is referenced in the RETURNING clause, which you have covered.

Reply:
thanks for catching that, I've updated the third column in the `INSERT` row to remove the check mark and replace with the `-` marker, and left the `INSERT ... RETURNING` case as is